README.md

```diff
@@ -121,7 +121,8 @@ See [Basic Concepts](docs/content/concepts/basic-concepts.md) for more details a
 FeatHub supports the following compute engines to execute feature ETL pipeline:
 - [Apache Flink 1.15](docs/content/engines/flink.md)
 - [Aapche Spark 3.3](docs/content/engines/spark.md)
-- Local processor, which is implemented using the Pandas library and computes features in the given Python process.
+- Local processor, which is implemented using the Pandas library and Spark local
```

Comment:
Let's not forcing users to always use pyspark to use local processor. How about this:

Local processor, which is implemented using the Pandas library and computes features in the given Python process. Additionally, if the feathub-nightly[spark] is installed, the Local processor can utilize pyspark local mode for accessing storages (e.g. HDFS) that it otherwise did not support.